src/main/resources/model/instance.json

```diff
@@ -135,11 +135,11 @@
           "showInResponse": [ "search"]
         },
         "date1": {
-          "index": "source",
+          "index": "standard",
           "showInResponse": [ "search" ]
         },
         "date2": {
-          "index": "source",
+          "index": "standard",
           "showInResponse": [ "search" ]
```

Comment:
Is it required to have it as full-text type? Could we modify the All-value processor to achieve the result without changing a type from source? There is more fields that have source type, so they will have same problem.